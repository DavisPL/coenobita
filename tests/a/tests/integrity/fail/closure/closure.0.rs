#![allow(unused_variables)]
#![allow(unused_assignments)]

fn main() {
    let f = |x: i32| x;

    #[cnbt::integrity({root}{root})]
    let y = f(5);
}