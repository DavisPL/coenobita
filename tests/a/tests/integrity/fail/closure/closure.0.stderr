error: expected {root}{root}, found {*}{*}
 --> tests/integrity/fail/closure/closure.0.rs:8:13
  |
8 |     let y = f(5);
  |             ^^^^
